frontend/lib/norent/homepage.tsx

```diff
@@ -1,19 +1,364 @@
 import React from "react";
 import Page from "../ui/page";
-import { Link } from "react-router-dom";
 import { NorentRoutes } from "./routes";
+import { CenteredPrimaryButtonLink } from "../ui/buttons";
+import { StaticImage } from "../ui/static-image";
+import { OutboundLink } from "../analytics/google-analytics";
+import { Link } from "react-router-dom";
+import { NorentLogo } from "./components/logo";
+
+type NorentImageType = "png" | "svg";
+
+export function getImageSrc(name: string, type?: NorentImageType) {
+  return `frontend/img/norent/${name}.${type || "svg"}`;
+}
+
+const checklistItems = [
+  "Build a letter using our free letter builder",
+  "Send your letter by email",
+  "Make a special request to your landlord",
+  "Send your letter by certified mail for free",
+  "Cite up-to-date legal ordinances in your letter",
+];
+
+const LandingPageChecklist = () => (
+  <div className="jf-space-below">
+    {checklistItems.map((checklistItem, i) => (
+      <article className="media" key={i}>
+        <div className="media-left">
+          <StaticImage
+            ratio="is-32x32"
+            src={getImageSrc("checkmark")}
+            alt="Check mark"
+          />
+        </div>
+        <div className="media-content">{checklistItem}</div>
+      </article>
```

Comment:
Hmm, semantically this actually seems like it should be a `<ul>`, which is  nice for screen readers because they actually read it aloud as a list.  I don't think it's that big a deal though, especially considering it'd take a lot of work to refactor the CSS to be a list.

But I think that in the context of how it's used, it makes sense to set `alt=""` on the image, because screen reader users will just want to hear the list of things they can do without hearing "check mark" between each one.  Or alternatively, set `alt="you can"` because in a way, that's what the check mark means semantically, and it will be read aloud in context as e.g. "you can build a letter using our free letter builder", "you can send your letter by email", etc, which should be very understandable to visually impaired users.

Reply:
I like the "You can" approach! Seems to be a happy middle ground